java/core/src/java/org/apache/orc/EncryptionVariant.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.orc;
+
+import java.io.IOException;
+import java.security.Key;
+
+/**
+ * Information about a key used for column encryption in an ORC file.
+ */
+public interface EncryptionVariant extends Comparable<EncryptionVariant> {
+
+  /**
+   * Get the encryption key for this column.
+   * @return the encryption key.
+   */
+  EncryptionKey getKey();
```

Comment:
use getColumnKey() to be more descriptive? Also the scope of these keys are stripe or file? Can it be explicitly called out?

Reply:
They are for the entire file, I'll update the name.

Reply:
More precisely, it is the key description so I updated the name to getKeyDescription.